ansible/inventory/group_vars/apps.yaml

```diff
@@ -206,6 +206,14 @@ chaos_monkey:
   root_vol_size: 20
   tag_name: AEM Chaos Monkey
   include_stack: true
+  termination_settings:
+    calendar_open_hour: '9'
+    calendar_close_hour: '15'
+    calendar_timezone: 'Australia/Sydney'
+    scheduler_frequency: '5'
```

Comment:
Rename this to `scheduler_frequency_in_minutes` . Consistent with other time-related configs, define the suffix `in_minutes`, `in_seconds`, or `in_hours`.